src/modules/order/order.controller.ts

```diff
@@ -29,8 +29,8 @@ import { AdminJwtAuthGuard } from '../auth/admin/guard/admin.jwt-auth.guard';
 import { JwtAuthGuard } from '../auth/user/guard/jwt-auth.guard';
 import { Role } from '../user/role.enum';
 import { RoleGuard } from '../user/role.guard';
-import { CartDto } from './dto/CartDto';
```

Comment:
should be removed. as far as I understand now we use CartWithPromocodeDto in every place, not CartDto.

Also we can just modify CartDto, add not required field promocode. It will require less changes